stable/mission-control/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.1.0] -  Jul 6, 2020
```

Comment:
@rahulsadanandan  can you bump it to 4.2.0 (we need to reserve 4.1.0 for 4.4.0 release)